lib/pbench/test/unit/server/query_apis/test_controllers_list.py

```diff
@@ -38,6 +38,33 @@ def test_non_accessible_user_data(self, client, server_config, pbench_token):
         )
         assert response.status_code == HTTPStatus.FORBIDDEN
 
+    def test_accessing_data_with_invalid_token(
+        self, client, server_config, pbench_token
+    ):
+        """
+        Test behavior when Authorization header does not have access to other user's data
+        """
+        # valid token logout
+        response = client.post(
+            f"{server_config.rest_uri}/logout",
+            headers=dict(Authorization="Bearer " + pbench_token),
+        )
+        assert response.status_code == 200
```

Comment:
You should use `HTTPStatus.OK` instead of the numeric literal here (and in datasets-list and datasets-details).